examples/cpp_api_inference/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.5)
+project (CAI)
+
+set (MIGRAPHX_FOLDER AMDMIGraphX)
+set (MIGRAPHX_BUILD ${MIGRAPHX_FOLDER}/build)
+
+set (CMAKE_CXX_STANDARD 14)
+
+include_directories(../../../${MIGRAPHX_FOLDER}/src/api/include)
+link_directories(../../../${MIGRAPHX_BUILD}/lib)
```

Comment:
The include directories are added with `migraphx_c`.

Reply:
Also, the `link_directories` should not be necessary.

Reply:
Removed